### ISSUE TITLE
Нормально - но часто нужно создать базу или Иблок до наката orm и модели.

### ISSUE THREAD

**Issue by AndyDune** [random123321/test#2]
https://github.com/random123321/test/blob/76eb33c10ec346b3db4af6a37b28c4ec530edd63/Version120180304184723.php#L17

**Comment by random123321** [comment]
Согласен, таблицу можно описать в отдельном классе или подключать файл напрямую. Для упрощения сделал через модуль
